manifest.toml

```diff
@@ -0,0 +1,76 @@
+packaging_format = 2
+
+id = "nextcloud"
+name = "Nextcloud"
+description.en = "Online storage, file sharing platform and various other applications"
+description.fr = "Stockage en ligne, plateforme de partage de fichiers et diverses autres applications"
+
+version = "27.1.2~ynh1"
+
+maintainers = ["kay0u"]
+
+[upstream]
+license = "AGPL-3.0"
+website = "https://nextcloud.com"
+demo = "https://demo.nextcloud.com/"
+admindoc = "https://docs.nextcloud.com/server/stable/admin_manual/"
+userdoc = "https://docs.nextcloud.com/server/latest/user_manual/en/"
+code = "https://github.com/nextcloud/server"
+cpe = "cpe:2.3:a:nextcloud:nextcloud"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = ["amd64", "arm64", "armhf"]
+multi_instance = true
+ldap = true
+sso = true
+disk = "50M"
+ram.build = "500M"
+ram.runtime = "512M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/nextcloud"
+
+    [install.admin]
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "You must activate 'Visitors' if you want to connect Nextcloud Desktop client to Nextcloud server. This can be changed later via the webadmin."
+    help.fr = "Vous devez activer 'Visiteurs' si vous souhaitez connecter le client Nextcloud Desktop au serveur Nextcloud. Cela peut être modifié ultérieurement via l'administrateur Web."
+    type = "group"
+    default = "visitors"
+
+    [install.user_home]
+    ask.en = "Access the users home folder from Nextcloud?"
+    ask.fr = "Accéder au dossier personnel des utilisateurs depuis Nextcloud ?"
+    type = "boolean"
+    default = false
+
+[resources]
+    [resources.system_user]
+
```

Comment:
cf a question today on the support chat, we may want to enable email in the future, to properly configure email :  

![nextcloudemail](https://github.com/YunoHost-Apps/nextcloud_ynh/assets/4533074/8291d368-6a09-4525-a141-773596dcc4b6)

(but i don't know much about nextcloud admin, maybe i'm mistaking)